config/rbac/submariner-operator/ocp_cluster_role.yaml

```diff
@@ -12,3 +12,9 @@ rules:
       - securitycontextconstraints
     verbs:
       - use
+  - apiGroups:
+      - config.openshift.io
+    resources:
+      - infrastructures
```

Comment:
In the addon code, I see that we are configuring explicit access to "infrastructures/status"
Please check if similar thing needs to be done even here.

https://github.com/stolostron/submariner-addon/blob/61ece25b8f65c6fa522b4f3216871c85f472b27b/deploy/config/rbac/cluster_role.yaml#L50

Reply:
Well, I checked it on OCP (ROKS) cluster and it works without explicit access to "infrastructures/status", 
also I can see that for example SRIOV operator [doesn't specify infrastructure/status](https://github.com/openshift/sriov-network-operator/blob/24856584100497eb402de673ab8fd37f25add796/deploy/clusterrole.yaml#L33) and [use fields from infra/status](https://github.com/openshift/sriov-network-operator/blob/24856584100497eb402de673ab8fd37f25add796/pkg/utils/cluster.go#L112)